aws/resource_aws_sagemaker_notebook_instance.go

```diff
@@ -0,0 +1,366 @@
+package aws
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/sagemaker"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"log"
+	"time"
```

Comment:
The  common pattern in this codebase is to separate stdlib imports from others, so these across this PR should be 

```go
"fmt"
"log"
"time"

"github.com/aws/aws-sdk-go/aws"
...
```

etc, eg: https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/resource_aws_api_gateway_api_key_test.go#L4

Reply:
FYI `goimports` is the tool that does this automatically (personally I have it setup in my editor on save): https://godoc.org/golang.org/x/tools/cmd/goimports

Reply:
Thank boths, I'll fix this today